core/node/rpc/service_test.go

```diff
@@ -4,7 +4,10 @@ import (
 	"bytes"
 	"context"
 	"crypto/tls"
+	"errors"
 	"fmt"
+	"github.com/river-build/river/core/node/base"
+	"go.uber.org/atomic"
```

Comment:
It seems regular `sync\atomic` has `Bool`?
If this package is to be used, then `go mod tidy` should be run.